README.adoc

```diff
@@ -1,8 +1,9 @@
 ifdef::env-github[]
-image::docs/images/Anakin.png[width="250", align="center"]
+image:docs/images/Anakin.png[width="250", align="center"]
 endif::[]
+
 ifndef::env-github[]
-image::images/Anakin.png[width="250", align="center"]
+image:docs/images/Anakin.png[width="200", align="center"]
```

Comment:
Should check with asciidoc. This one affects rendering of image on published version

Reply:
the file path for non-github and github is different because of a different relative root directory in the published version.